### DISCUSSION TITLE
Can I somehow switch off printing of that “[ble: reload]” message?

### DISCUSSION THREAD

**Discussion by mcepl** [akinomyoga/ble.sh#500] [Q&A] [answered]
It is cute the first couple of times, but the hundredth time not so much.

**Comment by akinomyoga** [comment, marked as answer]
You can set [`bleopt edit_marker=`](https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A74-Editing#user-content-bleopt-edit_marker) to disable similar markers. Some markers may not be turned off by this setting, or some may be separately configured, but at least `[ble: reload]` is affected by this setting. More details are [here](https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A74-Editing#user-content-bleopt-edit_marker).

**Reply by mcepl** [reply]
Thank you
